app/controllers/learning_units_controller.rb

```diff
@@ -4,7 +4,9 @@ class LearningUnitsController < ApplicationController
 
   def index; end
 
-  def show; end
+  def show
+    redirect_to learning_unit_resources_path(@learning_unit)
```

Comment:
I wouldn't do this, according to what we discussed yesterday.
I'd save this view as a learning_units' show page instead of a resources' index.